### DISCUSSION TITLE
How to build for arm64 on linux?

### DISCUSSION THREAD

**Discussion by Ang0426** [microsoft/msquic#3431] [Q&A]
It's failed when I build with "./scripts/build.ps1 -Arch arm64" on linux  and the error is :

error： "unsupported ARM architecture" 

but it can work build with "./scripts/build.ps1" or "./scripts/build.ps1 -Arch x86"

I want to use msquic on Android，

What do I need to do to compile out the arm64 msquic?

**Comment by Ang0426** [comment]
I resolved it with "./scripts/build.ps1 -Platform Android", now can used on Android

**Reply by nibanks** [reply]
I also recommend taking a look at https://github.com/microsoft/msquic/blob/main/.github/workflows/build.yml which builds for nearly all flavors. It should give you good examples.
